src/metabase/api/user_key_value.clj

```diff
@@ -0,0 +1,19 @@
+(ns metabase.api.user-key-value
+  (:require [compojure.core :refer [GET POST]]
+            [metabase.api.common :as api]
+            [metabase.models.user-key-value :as user-key-value]
+            [metabase.util.malli.schema :as ms]))
+
+(api/defendpoint POST "/"
```

Comment:
The RESTafarian in me suggests `PUT /:key` instead 😄

Reply:
Nice, good idea 👍